docs/octant-plugin-installation.md

```diff
@@ -91,25 +91,38 @@ You can follow the steps listed below to install octant and antrea-octant-plugin
     export KUBECONFIG=/etc/kubernetes/admin.conf
     ```
 
-3. Build antrea-octant-plugin.
+3. Get corresponding antrea-octant-plugin binary from [Release Assets](https://github.com/vmware-tanzu/antrea/releases/tag/v0.8.1)
+based on your environment and move the binary to OCTANT_PLUGIN_PATH.
 
-    ```
-    cd plugins/octant
-    make antrea-octant-plugin
-    ```
-
-4. Move antrea-octant-plugin to OCTANT_PLUGIN_PATH.
+    For example, you can get antrea-octant-plugin-linux-x86_64 if it matches your operating system and architecture.
 
     ```
+    wget -O antrea-octant-plugin https://github.com/vmware-tanzu/antrea/releases/download/v0.8.1/antrea-octant-plugin-linux-x86_64
+    # Make sure antrea-octant-plugin is executable, otherwise Octant can not find it.
+    chmod a+x antrea-octant-plugin
     # If you did not change OCTANT_PLUGIN_PATH, the default folder should be $HOME/.config/octant/plugins.
     mv antrea/plugins/octant/bin/antrea-octant-plugin $HOME/.config/octant/plugins/
```

Comment:
I think the source of the move should be `antrea-octant-plugin` based on the previous commands

Reply:
Right, thanks! Have updated.